test/utility/mockDashboardEmbed.ts

```diff
@@ -0,0 +1,63 @@
+import * as Wpmp from 'window-post-message-proxy';
+import * as Hpm from 'http-post-message';
+import * as Router from 'powerbi-router';
+import * as models from 'powerbi-models';
+import { mockAppSpyObj, mockApp } from './mockApp';
+
+export function setupDashboardEmbedMockApp(iframeContentWindow: Window, parentWindow: Window, logMessages: boolean, name: string = 'MockAppWindowPostMessageProxy'): Hpm.HttpPostMessage {
```

Comment:
Why did you choose to make a new mock app?

It looks like most of this code is copied form the original mock app and just changed the router to have a `/dashboard/load` handler.  It looks like this could be consolidated which would be less maintenance.  I think all you need is the code which adds the handler to the router.